### DISCUSSION TITLE
Is there a command line flag to enable shared memory in servo?

### DISCUSSION THREAD

**Discussion by jspanchu** [servo/servo#32256] [Q&A]
Hi,

I'm having trouble running a wasm application that creates a WebAssembly Memory instance with `shared: true`. Is there a flag or build option to enable this feature? Thanks.

Error:
```
shared memory is disabled at http://127.0.0.1:54346/index.js:367#16
```

**Comment by jdm** [comment]
That's probably controlled with https://github.com/servo/mozjs/blob/8603cbf35781ea8f2d57e4822a2b874f56a53914/mozjs-sys/mozjs/js/public/RealmOptions.h#L122-L141, and the default value there is https://github.com/servo/mozjs/blob/8603cbf35781ea8f2d57e4822a2b874f56a53914/mozjs-sys/mozjs/js/public/RealmOptions.h#L292 . We don't currently expose a flag to enable that in Servo at the moment.

**Reply by jdm** [reply]
Oh, and we explicitly set it to false in https://github.com/servo/servo/blob/168d43f24a18184afadc6aa8646537289c85860a/components/script/dom/bindings/interface.rs#L148, but that would be where the configuration flag would need to plug in.
